### PR TITLE
Elaborate MediaTrackCapabilities extensions

### LINKED ISSUE #93: MediaTrackCapabilities
We've extended MediaTrackConstraintSet and MediaTrackSettings. Should we add a partial dictionary for MediaTrackCapabilities as well?

I'm a little bit confused about similar information being in so many different places and I noticed we don't have this one.

### LINKED ISSUE #176: restrictOwnAudio is missing from MediaTrackSettings
* [MediaTrackConstraintSet](https://w3c.github.io/mediacapture-screen-share/#extensions-to-mediatrackconstraintset) currently covers `displaySurface`, `logicalSurface`, `cursor` and `restrictOwnAudio`.
* [MediaTrackSettings](https://w3c.github.io/mediacapture-screen-share/#extensions-to-mediatracksettings) neglects to mention `restrictOwnAudio`.

### PR DESCRIPTION
**Pull request by eladalon1983**
Fixes issues w3c/mediacapture-screen-share#93 and w3c/mediacapture-screen-share#176.

***
<a href="https://pr-preview.s3.amazonaws.com/eladalon1983/mediacapture-screen-share/pull/174.html" title="Last updated on Jun 3, 2021, 3:11 PM UTC (81705a5)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/mediacapture-screen-share/174/556ec65...eladalon1983:81705a5.html" title="Last updated on Jun 3, 2021, 3:11 PM UTC (81705a5)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -841,8 +842,73 @@ <h2>
...
+                that the user agent is capable of supporting for this track.
+              </p>
+            </dd>
+          <dt>
+             <dfn>restrictOwnAudio</dfn> of type {{boolean}}
+          </dt>
+          <dd>
+            <p>
+              MUST be the same value as is returned by {{MediaStreamTrack/getSettings()}},
+              rendering this property immutable from the application's viewpoint.
```
Comment (alvestrand) [inline index.html]: let's make this settable; if we can change it on the fly, return [true false]; if we can't, return one value.

Comment (jan-ivar) [inline index.html]: This shouldn't be immutable because it is not an inherent property of the source.

Comment (eladalon1983) [reply index.html]: Filed #180 for this.

Comment (eladalon1983) [reply index.html]: Filed #180 for this.

**Comment by eladalon1983** [comment]
I've removed the part that requires further discussion and filed issue w3c/mediacapture-screen-share-extensions#10 for it.

**Pull request merged**